src/py_ext_control_template/custom_timer.py

```diff
@@ -0,0 +1,66 @@
+from rospy.timer import *
+import rospy
+import threading
+
+class CustomTimer(threading.Thread):
```

Comment:
This CustomTimer is necessary for the pause() and restart() functions, right? I would suggest inheriting all properties from the rospy.Timer class and only adding this extra functionality, so we stick to the official ROS implementations as much as possible.